readme.md

```diff
@@ -221,4 +220,4 @@ The module contains a dataset as of `03/17/2017`. To re-generate the dataset or
 
 ## Contributing
 
-If you found a bug, want to propose a feature or feel the urge to complain about your life, feel free to visit [the issues page](https://github.com/juliuste/gemeindeverzeichnis/issues).
\ No newline at end of file
+If you found a bug, want to propose a feature or feel the urge to complain about your life, feel free to visit [the issues page](https://github.com/juliuste/gemeindeverzeichnis/issues).
```

Comment:
FYI this is probably your editor not complying with your `.editorconfig`. For Sublime Text, there's an "EditorConfig" plugin.